.clabot

```diff
@@ -48,7 +48,8 @@
     "christopherdro",
     "patrickworkman",
     "PJijin",
-    "adeelhasan"
+    "adeelhasan",
+    "easyrun42"
```

Comment:
Added @easyrun42 here and then @KosmosKey is added to another PR: https://github.com/unlock-protocol/unlock/pull/12581